pkg/actors/actors.go

```diff
@@ -226,7 +226,6 @@ func (a *actorsRuntime) callLocalActor(actorType, actorID, actorMethod string, d
 	lock.Lock()
 	defer lock.Unlock()
 
-	exists := a.actorInstanceExists(key)
```

Comment:
Remove the unused method actorInstanceExists now

Reply:
Already removed